lib/collection/src/operations/mod.rs

```diff
@@ -33,7 +33,7 @@ pub enum FieldIndexOperations {
     DeleteIndex(String),
 }
 
-#[derive(Debug, Deserialize, Serialize, JsonSchema, Clone)]
+#[derive(Debug, Deserialize, Serialize, Clone)]
 #[serde(rename_all = "snake_case")]
 #[serde(untagged)]
 pub enum CollectionUpdateOperations {
```

Comment:
I think openapi files need to be regenerated

Reply:
this structure is not actually used in rest, so there are no changes.